yarn.lock

```diff
@@ -3445,7 +3450,7 @@ joycon@^3.0.1:
   resolved "https://registry.yarnpkg.com/joycon/-/joycon-3.1.1.tgz#bce8596d6ae808f8b68168f5fc69280996894f03"
   integrity sha512-34wB/Y7MW7bzjKRjUKTa46I2Z7eV62Rkhva+KkopW7Qvv/OSWBqvkSY7vusOPrNuZcUG3tApvdVgNB8POj3SPw==
 
-js-beautify@^1.6.12:
+js-beautify@1.14.6, js-beautify@^1.6.12:
```

Comment:
Can you rebase and then run `yarn`? This shouldn't be included.

Reply:
shouldn't https://github.com/zenorocha/react-email/pull/107 be merged first?